bin/import_lingua_libre.py

```diff
@@ -177,12 +179,23 @@ def handle_args():
     parser.add_argument('--english-name', type=str, required=True, help='Enligh name of the language')
     parser.add_argument('--filter_alphabet', help='Exclude samples with characters not in provided alphabet')
     parser.add_argument('--normalize', action='store_true', help='Converts diacritic characters to their base ones')
+    parser.add_argument('--bogus-records', type=argparse.FileType('r'), required=False, help='Text file listing well-known bogus record to skip from importing, from https://lingualibre.fr/wiki/LinguaLibre:Misleading_items')
     return parser.parse_args()
 
 if __name__ == "__main__":
     CLI_ARGS = handle_args()
     ALPHABET = Alphabet(CLI_ARGS.filter_alphabet) if CLI_ARGS.filter_alphabet else None
 
+    bogus_regexes = []
+    for line in CLI_ARGS.bogus_records:
+        bogus_regexes.append(re.compile(line.strip()))
+
+    def record_filter(path):
```

Comment:
You don't use the return path of this function, so it'd make more sense if it returned a boolean instead.